### PR TITLE
More logging for refreshing feeds

### PR DESCRIPTION
**Pull request by jacksonh**
- Some debug
- Remove non-useful logging line
- Add a refresh context to make debugging rss jobs easier
- Add a refresh context to make debugging refresh runs easier

### CODE REVIEW

File: packages/api/src/jobs/save_page.ts [hunk 1]
```diff
@@ -353,11 +357,18 @@ export const savePageJob = async (data: Data, attemptsMade: number) => {
 
   try {
     const url = encodeURI(data.url)
+    console.log(`savePageJob: ${userId} ${url}`)
 
     // get the fetch result from cache
     const { title, content, contentType, readabilityResult } =
       await getCachedFetchResult(url)
 
+    if (!title || !content || !contentType || !readabilityResult) {
```
Comment (sywhb) [inline packages/api/src/jobs/save_page.ts:366]: content and readability can be null if content type is pdf

**Review by sywhb — CHANGES_REQUESTED** [review]

File: packages/api/src/jobs/save_page.ts [hunk 2]
```diff
@@ -382,30 +337,45 @@ export const savePageJob = async (data: Data, attemptsMade: number) => {
...
-      publishedAt,
-      source,
-      folder,
-    })
-    if (!apiResponse) {
-      throw new Error('error while saving page')
+    if (!content) {
+      throw new Error(
+        'Invalid SavePage job, fetch result missing required data'
+      )
```
Comment (sywhb) [inline packages/api/src/jobs/save_page.ts:343]: probably have to remove this check

File: packages/content-fetch/src/request_handler.ts [hunk 3]
```diff
@@ -138,7 +138,7 @@ export const contentFetchRequestHandler: RequestHandler = async (req, res) => {
     console.log('cacheFetchResult result', cacheResult)
 
     const jobs = await queueSavePageJob(savePageJobs)
-    console.log('save-page jobs queued', jobs.length)
+    console.log('save-page jobs queued', { jobs })
```
Comment (sywhb) [inline packages/content-fetch/src/request_handler.ts:141]: not sure if we need to log everything here if we want to reduce the log size

**Review by sywhb — APPROVED** [review]
LGTM 🚢

File: packages/api/src/jobs/save_page.ts (hunk 2 above)
Comment (jacksonh) [reply packages/api/src/jobs/save_page.ts:343]: I think we need it as we can't pass `originalContent` if its null.

File: packages/content-fetch/src/request_handler.ts (hunk 3 above)
Comment (jacksonh) [reply packages/content-fetch/src/request_handler.ts:141]: Yeah I'll remove this now

**Pull request merged**